src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,16 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
```

Comment:
There is a little more indentation than it is needed, two spaces for the if rows and 4 spaces for the return rows should be better.